client/my-sites/plans-features-main/wpcom-faq.jsx

```diff
@@ -17,6 +17,24 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<a href="https://wordpress.com/help" target="_blank" rel="noopener noreferrer" />
 		);
 
+	const themesAnswer = isEnabled( 'themes/premium' )
+		? translate(
+				"Yes! With the WordPress.com Business or eCommerce plan you can install any theme you'd like." +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Yes! With the WordPress.com Business or eCommerce plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/feature-list-with-premium-themes)